packages/@coorpacademy-components/src/molecule/brand-create-form/index.js

```diff
@@ -26,11 +26,16 @@ const BrandCreateForm = (props, context) => {
     </div>
   ) : (
     <div>
-      <Button
+      <ButtonLink
+        type="primary"
```

Comment:
we change the color from orange to primary blue, it's on purpose ?

Reply:
It is, all orange should be replaced with primary blue